src/fluree/db/util/ledger.cljc

```diff
@@ -32,9 +32,11 @@
    e.g., 'my-ledger' -> 'my-ledger:main'
          'my-ledger:branch' -> 'my-ledger:branch'"
   [ledger-alias]
-  (if (ledger-branch ledger-alias)
-    ledger-alias
-    (str ledger-alias ":" const/default-branch-name)))
+  (if (not-empty ledger-alias)
```

Comment:
Where does this get called from where we can receive a `nil` or `""` `ledger-alias`?

Reply:
Ok, I hope it doesn't come up, because in that case this function won't do what it says on the tin. Maybe we should throw in that case, since we've clearly got some bad inputs and we don't want to propagate a malformed ledger alias.